script/index.ts

```diff
@@ -55,16 +55,56 @@ function buildOptionsForLanguage(language: Language) {
       items: option.items
     };
   })
+  languageOptions["Beautifiers"] = {
```

Comment:
Shouldn't this be `beautifiers` instead of `Beautifiers`?

https://github.com/Unibeautify/unibeautify/blob/4f5a4b442ecf173c498435a5d0d16aa0499c5e5d/test/beautifier/pipeline.spec.ts#L13

Reply:
Hmm, I thought I saw in the interface it was capitalized when that was committed. But yea, you're right:  https://github.com/Unibeautify/unibeautify/blob/master/src/beautifier.ts#L308